doc/en/components/general-changelog-dv.md

```diff
@@ -20,6 +20,43 @@ All notable changes for each version of $ProductName$ are documented on this pag
 
 ## **$Platform$ {PackageVerChanges-21-2}**
 
+<!-- Blazor -->
+
+> [!NOTE]
+> 'Igb' is now required for all components and nested elements within each component. This change is mandatory change after upgrading
```

Comment:
replace "This change is mandatory change after upgrading" 
with "This API change was necessary to avoid ambiguity between Infragistics controls and 3rd party controls."